.github/workflows/goreleaser-fleet.yaml

```diff
@@ -95,19 +95,21 @@ jobs:
         with:
           subject-path: "dist/**"
 
-      # Get the commit hash so we can get image digests
-      - name: Get the short commit hash
-        id: commit
-        run: echo "short_commit=$(git rev-parse --short HEAD)" >> "$GITHUB_OUTPUT"
+      - name: Get tag
+        run: |
+          echo "TAG=$(git describe --tags |  sed -e "s/^fleet-//")" >> $GITHUB_OUTPUT
+        id: docker
 
       # Get the image digests from the goreleaser artifacts
       # Adapted from https://github.com/goreleaser/goreleaser/issues/4852#issuecomment-2122790132
       - name: Get image digests
         continue-on-error: true
         id: image_digests
         run: |
-          echo "digest_fleet=$(cat ./dist/artifacts.json | jq -r '.[]|select(.type == "Published Docker Image" and (.name | contains("fleetdm/fleet:${{ steps.commit.outputs.short_commit }}"))) | select(. != null)|.extra.Digest')" >> "$GITHUB_OUTPUT"
-          echo "digest_fleetctl=$(cat ./dist/artifacts.json | jq -r '.[]|select(.type == "Published Docker Image" and (.name | contains("fleetdm/fleetctl:${{ steps.commit.outputs.short_commit }}"))) | select(. != null)|.extra.Digest')" >> "$GITHUB_OUTPUT"
+          digest_fleet=$(cat ./dist/artifacts.json | jq -r 'first(.[]|select(.type == "Published Docker Image" and (.name == "fleetdm/fleet:${{ steps.docker.outputs.tag }}")) | select(. != null)|.extra.Digest)')
+          echo "digest_fleet=$digest_fleet" >> "$GITHUB_OUTPUT"
+          digest_fleetctl=$(cat ./dist/artifacts.json | jq -r 'first(.[]|select(.type == "Published Docker Image" and (.name  == "fleetdm/fleetctl:${{ steps.docker.outputs.tag }}")) | select(. != null)|.extra.Digest)')
+          echo "digest_fleetctl=$digest_fleetctl" >> "$GITHUB_OUTPUT"
```

Comment:
In [tests](https://github.com/fleetdm/fleet/actions/runs/13144208533/job/36678373921#step:7:1), I discovered that the artifacts.json file returned from goreleaser for our last release had multiple identical entries, which caused the `jq` parser to return multiple results separated by newlines.  I updated it to only take the first result, and to match the name _exactly_ rather than using `contains()`.